YouTube.sh

```diff
@@ -9,8 +13,6 @@
 # --yes-playlist == does download playlist if URL contains playlist
 # -f best == downloads the best quality video and audio
 
-user=`whoami`
```

Comment:
This isn't needed. You can just use $HOME or ~/ to refer to the home directory. You could also just literally have used `whoami` within the directory path itself.